shared/studio/tabs/dataview/dataInspector.tsx

```diff
@@ -951,3 +1002,80 @@ const ExpandedDataInspector = observer(function ExpandedDataInspector({
     </div>
   );
 });
+
+interface MobileDataInspectorProps {
+  expandedState: ExpandedInspector;
+}
+
+const MobileDataInspector = observer(function MobileDataInspector({
+  expandedState,
+}: MobileDataInspectorProps) {
+  const item = expandedState.getItems()?.[0] as ObjectLikeItem | undefined;
+
+  const state = useDataInspectorState().state;
+  const fields = state.fields || [];
+
+  const {navigate, currentPath} = useDBRouter();
+  const basePath = currentPath.join("/");
+
+  const closeExtendedView = () => {
+    state.setExpandedRowMobile(null);
+  };
+
+  return (
+    <div className={styles.mobileInspectorWindow}>
+      <div className={styles.header}>
+        <button onClick={closeExtendedView} className={styles.headerBtn}>
+          <BackIcon />
+        </button>
+
+        <p className={styles.title}>
+          {item ? item.data.__tname__ : `loading...`}
+        </p>
+        <button onClick={closeExtendedView} className={styles.headerBtn}>
+          <CrossIcon />
+        </button>
+      </div>
+      {item &&
+        fields.map((field, index) => {
+          const isMulti = field.multi;
+          const data = item.data;
+          const value = isMulti ? data[field.name].length : data[field.name];
+
+          const codec = state.dataCodecs?.[index];
+
+          return (
+            <div className={styles.field} key={field.name}>
+              <div className={styles.fieldHeader}>
+                <span className={styles.name}>{field.name}</span>
+                <span className={styles.type}>
+                  {`${isMulti ? "multi" : ""} ${field.typename}`}
+                </span>
+              </div>
+              {isMulti ? (
+                <button
+                  className={styles.linkObjName}
+                  onClick={() => {
+                    state.openNestedView(
+                      basePath,
+                      navigate,
+                      data.id,
+                      data.__tname__,
+                      field
+                    );
+                  }}
+                >
+                  {field.typename.split("::")[1]}
```

Comment:
This doesn't work for types in nested modules. You need to get the last part:
```suggestion
                  {field.typename.split("::").pop()}
```